package.json

```diff
@@ -302,6 +310,11 @@
 					"when": "view == gerrit:changeExplorer && viewItem =~ /gerritchange/",
 					"group": "change@1"
 				},
+				{
+					"command": "gerrit.checkoutBranch",
+					"when": "view == gerrit:changeExplorer && viewItem =~ /gerritchange/",
+					"group": "git@1"
```

Comment:
I'd say this command takes priority over the other change commands so it should be at the top. Since this is sorted alphabetically, can you change the group name to something that comes before `change@1`? I don't really care if it's `achange@1` or something.